### PR TITLE
S Corp Gacha Changes

### PR DESCRIPTION
**Pull request by tile4**
## About The Pull Request
Changes some things about the S corp Gacha. I added a very rare tab with some stronger gear, as well as added more shrimp gear. I also made two different shrimp related items for the gacha. Also, considering how none of the stuff is really that good, I reduced S Corp Gacha cost to 1 Box per instead of 1.5 

## Why It's Good For The Game
Adds more variety to S corp outlets

## Changelog
:cl:
add: Added shrimp mutation toxin
tweak: Added more things to SCorp Outlet.
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_items/refinery/crates/corporation.dm [hunk 1]
```diff
@@ -161,10 +163,20 @@
...
 
+	veryrareloot = list(
+		/obj/item/grenade/spawnergrenade/shrimp/super,
+		/obj/item/grenade/spawnergrenade/shrimp/hostile,
+		/obj/item/reagent_containers/pill/shrimptoxin,
+	)
+
+	cosmeticloot = list(
+		/mob/living/simple_animal/hostile/shrimp,
+		/mob/living/carbon/human/species/shrimp,
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/refinery/crates/corporation.dm:178]: This I would actually remove.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Just one small thing

File: ModularTegustation/tegu_items/refinery/crates/corporation.dm (hunk 1 above)
Comment (tile4) [reply ModularTegustation/tegu_items/refinery/crates/corporation.dm]: There we go, did what you told me to so all should be good.

**Review by Kitsunemitsu — APPROVED** [review]
Ok this is Epic

**Pull request merged**